src/main/java/com/scottlogic/practice/mealscheduler/Controllers/MealController.java

```diff
@@ -30,4 +32,13 @@ public Meal CreateMeal(@RequestBody Meal meal) {
         return mealService.CreateMeal(meal);
     }
 
+    @DeleteMapping(value = "/{id}")
+    @Timed("DeleteMeal")
+    public ResponseEntity<Integer> DeleteMeal(@PathVariable Integer id) {
+        var result = mealService.DeleteMeal(id);
+
+        return result.map(
+                deleteId -> new ResponseEntity<>(deleteId, HttpStatus.OK))
+                .orElseGet(() -> new ResponseEntity<>(HttpStatus.NOT_FOUND));
```

Comment:
would
```
 return result.map(
                deleteId -> ResponseEntity.ok(deleteId))
                .orElseGet(() -> ResponseEntity.notFound());
```
be more readable ?

Reply:
Oh I really like this! Didn't know these static methods existed, thanks!